.devcontainer/devcontainer.json

```diff
@@ -29,7 +29,10 @@
     "matklad.rust-analyzer",
     "tamasfe.even-better-toml",
     "serayuzgur.crates",
-    "fermyon.autobindle"
+    "fermyon.autobindle",
+    "golang.Go",
+    "grain-lang.vscode-grain",
+    "saulecabrera.asls"
```

Comment:
This appears to be the more active VS Code extension from Shopify: https://github.com/Shopify/vscode-as

Reply:
Is this preferred for AssemblyScript vs. the regular JavaScript / TypeScript extension?

Reply:
Not sure, I can reach out to folks in the AssemblyScript Discord and see what they've been using/prefer.

Reply:
Actually, it probably makes sense to just use the regular JavaScript / TypeScript extension by Microsoft (https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-next). If there's a recommendation from the AS core team, I can open up a follow-up.

Reply:
That is configured by default anyway.
Let's see if using the AssemblyScript tooling is an improvement, there are quite a things not caught by the JS language server when writing AssemblyScript.